pkg/ccl/backupccl/schedule_exec.go

```diff
@@ -99,19 +99,33 @@ func (e *scheduledBackupExecutor) executeBackup(
 	}
 	backupStmt.AsOf = tree.AsOfClause{Expr: endTime}
 
-	log.Infof(ctx, "Starting scheduled backup %d", sj.ScheduleID())
-
 	// Invoke backup plan hook.
 	hook, cleanup := cfg.PlanHookMaker(ctx, "exec-backup", txn.KV(), sj.Owner())
 	defer cleanup()
 
+	planner := hook.(sql.PlanHookState)
+	currentClusterID := planner.ExtendedEvalContext().ClusterID
+	currentDetails := sj.ScheduleDetails()
+	if currentClusterID != currentDetails.ClusterID {
```

Comment:
and currentDetails.ClusterID non-empty right? for backwards compatibility?

Reply:
oooof. nice freaking catch. fixed.